src/userobjects/PKAGeneratorBase.C

```diff
@@ -31,23 +33,31 @@ PKAGeneratorBase::setPosition(MyTRIM_NS::IonBase & ion) const
 }
 
 int
-PKAGeneratorBase::ionTag(const std::vector<Real> & rasterizer_Z, const std::vector<Real> & rasterizer_m, Real Z, Real m) const
+PKAGeneratorBase::ionTag(const MyTRIMRasterizer::PKAParameters & pka_parameters,
+                         Real Z,
```

Comment:
should we make Z an integer here?

Reply:
We could, but then we should make it integer in a whole bunch more places